build.gradle

```diff
@@ -26,30 +26,32 @@ plugins {
   id 'net.researchgate.release' version '2.8.1'
 }
 
+def repositoryName = "EthSigner"
```

Comment:
nit: EthSigner would be the product name and EthSigner.toLowerCase() would be repository name

Reply:
done - nicer.